workflow-templates/lint-php-cs.yml

```
@@ -42,7 +42,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Install dependencies
-        run: composer i
+        run: |
+          composer remove nextcloud/ocp --dev
+          composer i
```

shouldn't
```suggestion
          composer i --no-dev
```

be enough for code style?

---

`php-cs-fixer` not a development dependency?

---

Arg :speak_no_evil: